### DISCUSSION TITLE
New Name Idea(s): BIG Engine

### DISCUSSION THREAD

**Discussion by F35H** [UPBGE/upbge#1611] [Announcements & others]
This may seem trivial, but I have a few ideas for a new name for this project. I assumed it would eventually be required for a few reasons. Firstly, UPBGE isn't an easy to decipher acronym. It isn't easy to say, and even when the project is explained, most people are not going to understand UP stands for Uchronia Project. The best thing is that it's SEO [Search Engine Optimized].
Secondly, associating with the original Blender Game Engine is no longer really needed and possibly detrimental. The engine historically had a negative reputation, and most of the niche fan group who appreciated it have either moved on or have had their fill with the project. Changing the name would disassociate it from being a fan start-up project allowing it to become more of its own engine as well.

Because of these reasons, I propose a few names:

BIG Engine - Blender Integrated Game Engine
U-BIG Engine - Uchronia Blender Integrated Game Engine

What are your opinions?

**Comment by rpaladin** [comment]
The name "UPBGE" is meant to suggest plural words I believe, hence "UP" + "BGE" looks more the eye of promoting or an upgrade of the Blender Game Engine.

Additionally, the word "Uchronia" wasn't just randomly picked, but was carefully picked to indict somewhat an `alternate timeline`.
https://en.wikipedia.org/wiki/Uchronia

**Reply by F35H** [reply]
That's a good point. I honestly thought it referred to something like a name.

However, that doesn't really provide a counter to the main points. In fact, it possibly validates it even more.
